app/v2/models/errors/UpstreamError.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.errors
+
+import uk.gov.hmrc.http.UpstreamErrorResponse
+
+// Used to ensure we avoid breaking changes with http verbs
```

Comment:
What are the breaking changes?

Reply:
It's more future proofing.

The biggest issue I faced writing this PR was how Verbs populates the `UpstreamErrorResponse` object - it wraps the message body in its own text, meaning we would have to parse out the text, which is controlled by the Verbs team so it'd be foolish to assume it wouldn't change. 

Now, I could have used `UpstreamErrorResponse` instead in this case because it's not this object that makes the text transformation, but again, as an object controlled by verbs, I didn't want to rely on that always being the case. Hence, we have our own object which we know won't change.